src/convertToObject.js

```diff
@@ -12,7 +12,17 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+
+  sourceString.split(';').forEach(item => {
+    const correctString = item.split(':').map(word => word.trim(''));
```

Comment:
trim() method expects 0 arguments.
```suggestion
    const correctString = item.split(':').map(word => word.trim());
```